src/library/styles/components/_input-radio-button-group.scss

```diff
@@ -1,39 +1,48 @@
+/* stylelint-disable scss/selector-nest-combinators */
+
+@mixin hover-reset {
+  &:hover {
+    + .bsds-input-radio-label {
+      box-shadow: none;
+      border-color: transparent;
+    }
+  }
+}
+
 .bsds-fieldset-button-group {
   @extend .bsds-fieldset;
   flex-flow: row wrap;
   gap: space(lg);
 
-  &:not([aria-invalid='true']) {
-    gap: space(xs) space(lg);
-  }
-
   .bsds-legend {
     @include font-family-base-regular;
 
     ~ .bsds-field {
       width: max-content;
 
       &:not(:first-child) {
-        min-width: space(4x);
         margin-top: space(lg);
       }
     }
   }
 
   .bsds-input-radio {
     + .bsds-input-radio-label {
-      @include button-default;
-      @include font-family-base-regular;
-      outline: space(xxs) solid $neutral-40;
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      min-width: space(8x);
       border-radius: space(3x);
-      border: $border-width-xs solid transparent;
+      border-width: space(xxs);
+      border-style: solid;
+      border-color: inherit;
       padding: space(md) space(3x);
+      background: $neutral-100;
       color: $text-color-subtle;
-
-      &:hover,
-      &:active {
-        outline: space(xxs) solid $action-primary-20;
-      }
+      font-size: $ffs-body-assertive;
```

Comment:
Should this be `@include font-size($ffs-body-assertive)`?

Reply:
this one is the non-tokens library var for fixed font size